### PR TITLE
Jira workflow

### PR DESCRIPTION
**Pull request by marek-saji**
- ⛔ Blocked by https://github.com/verkstedt/jira-integration-action/pull/4 (changes input names to use “status” instead of “list”)
- 🍷 Pairs with https://github.com/verkstedt/.github/pull/3

# Why?

Closes https://verkstedt.atlassian.net/browse/VIP-3

# What?

Introduces workflow similar to “trello”, but for “jira”. After our project migrate to “jira”, we can remove “trello” from the repository. [`trello-integration-action`](https://github.com/verkstedt/trello-integration-action) is already archived.

### CODE REVIEW

File: .github/workflows/jira.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Verify workflow setup
+        run: |
+          exit_code=0
+
+          if [ -z "${{ vars.JIRA_DOMAIN }}" ]
+          then
+            echo "::error::Missing var JIRA_DOMAIN. Set it to the origin you are accessing jira on, e.g. https://example.atlassian.net. It’s a good idea to set it on organisation level."
+            exit_code=78
```
Comment (stepahn) [inline .github/workflows/jira.yaml:18]: TIL `man sysexits`

**Review by stepahn — APPROVED** [review]

File: .github/workflows/jira.yaml (hunk 1 above)
Comment (marek-saji) [reply .github/workflows/jira.yaml:18]: only thing you are risking is people thinking you are trying to be funny, when you are using EX_UNAVAILABLE.

Comment (stepahn) [reply .github/workflows/jira.yaml:18]: 😂

Comment (stepahn) [reply .github/workflows/jira.yaml:18]: but alas it has probably no mischevious background…

```c
#define EX__BASE       64  /* Base value for error messages */

/* Below are the exit codes relative to EX__BASE */
#define EX_USAGE       (EX__BASE + 0)  /* Command line usage error */
#define EX_DATAERR     (EX__BASE + 1)  /* Data format error */
#define EX_NOINPUT     (EX__BASE + 2)  /* Cannot open input */
#define EX_NOUSER      (EX__BASE + 3)  /* User unknown */
#define EX_NOHOST      (EX__BASE + 4)  /* Host unknown */
#define EX_UNAVAILABLE (EX__BASE + 5)  /* Service unavailable */
```

Comment (stepahn) [reply .github/workflows/jira.yaml:18]: though it's not defined like this in the original… but it has a hell of a commit timestamp 😳 

Screenshot 2024-10-30 at 13 03 59

Comment (marek-saji) [reply .github/workflows/jira.yaml:18]: > has a hell of a commit timestamp

Linux man for `sysexits.h` says:

> HISTORY
>       The <sysexits.h> file appeared in 4.0BSD for use by the deliverymail utility, later renamed to sendmail(8).

So it originates from [1980 or earlier](https://en.wikipedia.org/wiki/History_of_the_Berkeley_Software_Distribution#4BSD). :slightly_smiling_face:

**Pull request merged**